fullcontact/api/base/enrich_base.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module serves the base class for all
+FullContact Enrich API request classes.
+"""
+
+from abc import ABCMeta, abstractmethod
+from concurrent.futures import Future
+
+from .base import ApiBase
+
+
+class EnrichBase(ApiBase, metaclass=ABCMeta):
```

Comment:
For consistent API class name, name it EnrichBaseApi.

Reply:
Done 👍